src/zeroconf/__init__.py

```diff
@@ -78,7 +78,7 @@
     add_multicast_member,
     autodetect_ip_version,
     create_sockets,
-    get_all_addresses,
```

Comment:
These methods are part of the public api so we need to keep it around to avoid a breaking change.

Reply:
I think it makes sense to deprecate calling them externally since they can use `ifaddr` now

Reply:
Should can/I do something in this PR to deprecate them?

Reply:
Something like this when called from the public interface (may need to make a protected version and use it internally to avoid the warning):

`warnings.warn("XXX is deprecated, and will be removed in a future version", DeprecationWarning, stacklevel=2)`

Reply:
We should probably tell them what to do instead in the text otherwise they will open issues asking what to do instead.